apps/exploits/api.py

```diff
@@ -344,6 +345,17 @@ def get(self, request):
         )
 
 
+class ExploitsReportData(ListAPIView):
+    """Export only the data required to generate the exploits report"""
+
+    serializer_class = ExploitReportDataSerializer
+    queryset = (
+        Exploit.objects.filter(source="CISA")
```

Comment:
Change to `Exploit.objects.filter(Q(source="CISA") | Q(source="Metasploit"))` as we are quickly approaching ability to handle Metasploit.

Reply:
You are right and that is actually what we did in the end.